Description: Disable examples, with features and deps solely used in them
Forwarded: not-needed
Last-Update: 2024-06-18
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -52,5 +52,5 @@
 
-[dependencies.cpuprofiler]
-version = "0.0.4"
-optional = true
+#[dependencies.cpuprofiler]
+#version = "0.0.4"
+#optional = true
 
@@ -72,2 +72,2 @@
-[dev-dependencies.lazy_static]
-version = "1.5"
+#[dev-dependencies.lazy_static]
+#version = "1.5"
@@ -80,2 +80,2 @@
-profiling = []
-verbose_fuse_test_log = []
+#profiling = []
+#verbose_fuse_test_log = []
--- a/examples/cntrfs.rs
+++ b/examples/cntrfs.rs
@@ -1,2 +1,5 @@
-use cntr::fs::{CntrFs, CntrMountOptions};
+fn main () {
+	eprintln!("Debian: disabled example");
+}
+/*use cntr::fs::{CntrFs, CntrMountOptions};
 
@@ -114,2 +117,2 @@
     PROFILER.lock().unwrap().stop().unwrap();
-}
+}*/
